integreat_cms/cms/forms/translations/translations_management_form.py

```diff
@@ -26,3 +76,52 @@ class Meta:
             "machine_translate_events",
             "machine_translate_pois",
         ]
+
+    def __init__(self, *args, **kwargs):
+        r"""
+        Initialize translations management form
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+        """
+        super().__init__(*args, **kwargs)
+
+        self.fields[
+            "machine_translatable_languages"
+        ].queryset = self.instance.language_tree_nodes.filter(active=True).order_by(
+            "language"
+        )
+        self.initial[
+            "machine_translatable_languages"
+        ] = self.instance.language_tree_nodes.filter(machine_translation_enabled=True)
+
+    def save(self, commit=True):
+        """
+        This method extends the default ``save()``-method of the base :class:`~django.forms.ModelForm` to set attributes
+        which are not directly determined by input fields.
+
+        :param commit: Whether or not the changes should be written to the database
+        :type commit: bool
+
+        :return: The saved region object
+        :rtype: ~integreat_cms.cms.models.regions.region.Region
+        """
+
+        # Save CustomModelForm
+        region = super().save(commit=commit)
+
+        # Since cleaned_data only contains checked checkboxes, iterate over all region languages
+        for language in self.instance.language_tree_nodes.filter(active=True):
+            if language in self.cleaned_data["machine_translatable_languages"]:
+                language.machine_translation_enabled = True
+                language.save()
+            elif language.mt_provider:
+                # do not alter languages without a MT provider, i.e. those that
+                # were unchecked and disabled in CustomCheckboxSelectMultiple
+                language.machine_translation_enabled = False
+                language.save()
```

Comment:
```suggestion
        # Iterate over all machine translatable languages and toggle them based on the checkbox value
        for language in self.fields["machine_translatable_languages"].queryset:
            language.machine_translation_enabled = (
                language in self.cleaned_data["machine_translatable_languages"]
            )
            language.save()
```
If we correctly limit the queryset to all languages which are possible, we can let Django handle the validation and only need to care about saving here.